testsupport/wait/member.go

```diff
@@ -108,7 +109,7 @@ func (a *MemberAwaitility) printUserAccountWaitCriterionDiffs(actual *toolchainv
 		buf.WriteString("\n----\n")
 		buf.WriteString("diffs:\n")
 		for _, c := range criteria {
-			if !c.Match(actual) {
+			if !c.Match(actual) && c.Diff != nil {
```

Comment:
same question as above. Maybe we should add this nil-check in all other places?

Reply:
Yes, I had the same thought. I'll handle it in a separate PR. Noted.